util/template_utils.go

```diff
@@ -100,6 +100,14 @@ const FilenameDiffOutput = `
 {{.Diff}}
 `
 
+const SizeDiffOutput = `
+-----{{.DiffType}}-----
+
+Size differences between {{.Image1}} and {{.Image2}}:{{if not .Diff}} None{{else}}
+LAYER	SIZE1	SIZE2{{range .Diff}}{{"\n"}}{{.Name}}	{{.Size1}}	{{.Size2}}{{end}}
```

Comment:
should this be `IMAGE` instead of `LAYER`?

Reply:
Please see this comment: https://github.com/GoogleContainerTools/container-diff/pull/256#discussion_r216526324